### ISSUE TITLE
Unknown Firewall (753181e39)

### ISSUE THREAD

**Issue by WhatWaf-Firewalls** [Ekultek/WhatWaf#625]
WhatWaf version: `1.6.11`
Running context: `whatwaf.py -u ************************************************************************************************************************************************************************** -p ids[0][product_id][to]=)))%20OR%20(SELECT%201%20UNION%20SELECT%202%20FROM%20DUAL%20WHERE%201=1)%20--%20-`
Fingerprint:
```
<!--
GET http://www.simpsonspremium.com HTTP/1.1
Status code: 404
Server: nginx
Date: Fri, 29 Nov 2019 18:02:27 GMT
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: close
X-Powered-By: PHP/7.2.25
Link: <https://www.simpsonspremium.com/wp/index.php?rest_route=/>; rel="https://api.w.org/"
X-Magento-Cache-Control: max-age=86400, public, s-maxage=86400
X-Magento-Cache-Debug: MISS
X-Magento-Tags: store,cms_b,cms_b_block_header_top_links,cms_b_block_header_top_left,cms_b_block_header_top_right,cms_b_ups_block,cms_b_footer-ranges-dogs,cms_b_footer-ranges-dogs-2,cms_b_footer-ranges-cats,cms_b_block_footer_primary_bottom_left,cms_b_block_footer_primary_bottom_right,cms_b_block_footer_column1,cms_b_block_footer_column2,cms_b_block_footer_column3,cms_b_block_footer_column4,cms_b_block_footer_column5,cms_b_block_footer_column6,cms_b_block_footer_payment,cat_c,store_group,cms_b_block_nav_links,cms_b_menu_useful_info,cms_b_advice_centre_l,cms_b_advice_centre_r,cms_p_1,cms_b_footer_contact,cms_b_footer-policies,FPC
Pragma: no-cache
Cache-Control: max-age=0, must-revalidate, no-cache, no-store
Expires: Thu, 29 Nov 2018 18:02:27 GMT
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
X-Frame-Options: SAMEORIGIN
X-UA-Compatible: IE=edge
-->
<!DOCTYPE doctype html>

<html lang="en">
<head>
<script>
    var BASE_URL = 'https://www.simpsonspremium.com/';
    var require = {
        "baseUrl": "https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB"
    };
</script>
<meta charset="utf-8"/>
<meta content="Page description" name="description"/>
<meta content="Page keywords" name="keywords"/>
<meta content="NOINDEX,NOFOLLOW" name="robots"/>
<meta content="404" name="title"/>
<meta content="width=device-width, initial-scale=1, maximum-scale=1.0" name="viewport"/>
<meta content="telephone=no" name="format-detection"/>
<title>404</title>
<link href="https://www.simpsonspremium.com/pub/static/version1574949831/_cache/merged/85216fb387b4b3070e990a5d70b80410.css" media="all" rel="stylesheet" type="text/css"/>
<link href="https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/css/styles-l.css" media="screen and (min-width: 768px)" rel="stylesheet" type="text/css"/>
<link href="https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/css/print.css" media="print" rel="stylesheet" type="text/css"/>
<script src="https://www.simpsonspremium.com/pub/static/version1574949831/_cache/merged/904492b5f5ae2777dee059a601b2a07e.js" type="text/javascript"></script>
<link href="https://www.simpsonspremium.com/pub/media/favicon/default/favicon-194x194.png" rel="icon" type="image/x-icon"/>
<link href="https://www.simpsonspremium.com/pub/media/favicon/default/favicon-194x194.png" rel="shortcut icon" type="image/x-icon"/>
<!-- Google Tag Manager by MagePal -->
<script type="text/javascript">
    window.dataLayer = window.dataLayer || [];
</script>
<script type="text/x-magento-init">
    {
        "*": {
            "magepalGtmDatalayer": {
                "isCookieRestrictionModeEnabled": 0,
                "currentWebsite": 1,
                "cookieName": "user_allowed_save_cookie",
                "dataLayer": "dataLayer",
                "accountId": "GTM-M5KKTGL",
                "data": [{"ecommerce":{"currencyCode":"GBP"},"pageType":"cms_noroute_index","list":"other"}],
                "isGdprEnabled": 0,
                "gdprOption": 0            }
        }
    }
</script>
<!-- End Google Tag Manager by MagePal -->
<link href="https://www.simpsonspremium.com/asset/dynamic/assets/m/iult/f/cfg.css/" media="all" rel="stylesheet" type="text/css"/>
</head>
<body class="cms-no-route lang-uk base cms-noroute-index page-layout-1column" data-container="body" data-mage-init='{"loaderAjax": {}, "loader": { "icon": "https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/images/loader-2.gif"}}'>
<script>
    require.config({
        map: {
            '*': {
                wysiwygAdapter: 'mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter'
            }
        }
    });
</script>
<script>
        require.config({
            deps: [
                'jquery',
                'mage/translate',
                'jquery/jquery-storageapi'
            ],
            callback: function ($) {
                'use strict';

                var dependencies = [],
                    versionObj;

                $.initNamespaceStorage('mage-translation-storage');
                $.initNamespaceStorage('mage-translation-file-version');
                versionObj = $.localStorage.get('mage-translation-file-version');

                
                if (versionObj.version !== '19c060f727aab60a01376bbbeb0c61515778bd0c') {
                    dependencies.push(
                        'text!js-translation.json'
                    );

                }

                require.config({
                    deps: dependencies,
                    callback: function (string) {
                        if (typeof string === 'string') {
                            $.mage.translate.add(JSON.parse(string));
                            $.localStorage.set('mage-translation-storage', string);
                            $.localStorage.set(
                                'mage-translation-file-version',
                                {
                                    version: '19c060f727aab60a01376bbbeb0c61515778bd0c'
                                }
                            );
                        } else {
                            $.mage.translate.add($.localStorage.get('mage-translation-storage'));
                        }
                    }
                });
            }
        });
    </script>
<script type="text/x-magento-init">
    {
        "*": {
            "mage/cookies": {
                "expires": null,
                "path": "/",
                "domain": ".www.simpsonspremium.com",
                "secure": false,
                "lifetime": "3600"
            }
        }
    }
</script>
<noscript>
<div class="message global noscript">
<div class="content">
<p>
<strong>JavaScript seems to be disabled in your browser.</strong>
<span>For the best experience on our site, be sure to turn on Javascript in your browser.</span>
</p>
</div>
</div>
</noscript>
<!-- Google Tag Manager by MagePal -->
<noscript><iframe height="0" src="//www.googletagmanager.com/ns.html?id=GTM-M5KKTGL" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<!-- End Google Tag Manager by MagePal -->
<div class="page-wrapper"><div class="header-container header-mobile page-header" id="header-container">
<div class="header-container2" id="top">
<div class="header-container3">
<!-- Part of the header displayed only in mobile mode -->
<div class="header-m-container">
<div class="header-m-top-container">
<div class="header-m-top header container clearer">
<div class="inner-container">
</div> <!-- end: inner-container -->
</div> <!-- end: header-m-top -->
</div> <!-- end: header-m-top-container -->
<div class="header-m-primary-container">
<div class="header-m-primary header container">
<div class="inner-container">
<!-- Skip links -->
<div class="skip-links-wrapper skip-links--4" id="skip-links">
<!-- Mobile logo -->
<div class="logo-wrapper--mobile">
<a class="logo" href="https://www.simpsonspremium.com/" title="Simpsons Premium">
<strong>Simpsons Premium</strong>
<img alt="Simpsons Premium" src="https://www.simpsonspremium.com/pub/media/logo/default/logo.png"/>
</a>
</div>
<div class="clearer after-mobile-logo"></div>
<div class="icons-wrapper--mobile">
<div class="skip-link skip-lang" id="lang-switcher-wrapper-mobile">
<div class="dropdown-block filled-heading dropdown-block-trigger lang-switcher lang-switcher-mobile" id="switcher-language-mobile">
<div class="dropdown-heading">
<span class="inner-heading view-uk" title="Language">
<span class="dropdown-icon"> </span>
<span class="label">English</span>
<span class="caret"></span>
</span>
</div>
<ul class="dropdown-content">
<li class="view-it switcher-option">
<a href="/it"><span class="label dropdown-icon"> </span>Italian</a>
</li>
<li class="view-fr switcher-option">
<a href="/fr"><span class="label dropdown-icon"> </span>French</a>
</li>
<li class="view-nl switcher-option">
<a href="/nl"><span class="label dropdown-icon"> </span>Dutch</a>
</li>
<li class="view-be switcher-option">
<a href="/be"><span class="label dropdown-icon"> </span>Belgian</a>
</li>
</ul>
</div>
</div>
<div class="skip-link skip-nav" data-skiptarget="#header-nav">
<span class="icon ic ic-menu"></span>
<span class="label">Menu</span>
</div>
<div class="skip-link skip-search" data-skiptarget="#header-search">
<span class="icon ic ic-search"></span>
<span class="label">Search</span>
</div>
</div>
<div class="clearer after-icons"></div>
<div id="mini-cart-marker-mobile"></div>
<div class="minicart-wrapper mini-cart filled-heading" data-block="minicart" id="minicart"> <div class="mini-cart-heading dropdown-heading skip-link skip-cart action showcart counter-over-" data-bind="scope: 'minicart_content'" data-skiptarget="#header-cart">
<span class="inner-heading">
<span class="icon ic ic-cart" style="font-size: 20px;"></span>
<span class="counter qty empty ic ic-char ib ib-size-s ib-rounded" data-bind="css: { empty: !!getCartParam('summary_count') == false }, blockLoader: isLoading">
<span class="counter-number"><!-- ko text: getCartParam('summary_count') --><!-- /ko --></span>
<span class="counter-label">
<!-- ko if: getCartParam('summary_count') -->
<!-- ko text: getCartParam('summary_count') --><!-- /ko -->
<!-- ko i18n: 'items' --><!-- /ko -->
<!-- /ko -->
</span>
</span>
<span data-bind="html: getCartParam('subtotal')"></span>
<span class="label">Basket</span>
<span class="caret"></span>
</span>
<a href="/checkout" id="user-menu-checkout">
<button class="action">Checkout <i class="caret-right"> </i></button>
</a>
</div>
<div class="mini-cart-content dropdown-content skip-content skip-content--style block-cart block block-minicart empty" data-mage-init='{"dropdownDialog":{
                "triggerEvent": "click mouseenter",
                "appendTo": "[data-block=minicart]",
                "triggerTarget": ".showcart",
                "timeout": "200",
                "closeOnMouseLeave": true,
                "closeOnEscape": true,
                "parentClass": "active open",
                "buttons": []}}' data-role="dropdownDialog" id="header-cart">
<div data-bind="scope: 'minicart_content'" id="minicart-content-wrapper">
<!-- ko template: getTemplate() --><!-- /ko -->
</div>
</div>
<script>
        window.checkout = {"shoppingCartUrl":"https:\/\/www.simpsonspremium.com\/checkout\/cart\/","checkoutUrl":"https:\/\/www.simpsonspremium.com\/checkout\/","updateItemQtyUrl":"https:\/\/www.simpsonspremium.com\/checkout\/sidebar\/updateItemQty\/","removeItemUrl":"https:\/\/www.simpsonspremium.com\/checkout\/sidebar\/removeItem\/","imageTemplate":"Magento_Catalog\/product\/image_with_borders","baseUrl":"https:\/\/www.simpsonspremium.com\/","minicartMaxItemsVisible":3,"websiteId":"1","maxItemsToDisplay":10,"storeId":"1","customerLoginUrl":"https:\/\/www.simpsonspremium.com\/customer\/account\/login\/","isRedirectRequired":false,"autocomplete":"off","captcha":{"user_login":{"isCaseSensitive":false,"imageHeight":50,"imageSrc":"","refreshUrl":"https:\/\/www.simpsonspremium.com\/captcha\/refresh\/","isRequired":false,"timestamp":1575050547}}};
    </script>
<script type="text/x-magento-init">
    {
        "[data-block='minicart']": {
            "Magento_Ui/js/core/app": {"components":{"minicart_content":{"children":{"subtotal.container":{"children":{"subtotal":{"children":{"subtotal.totals":{"config":{"display_cart_subtotal_incl_tax":1,"display_cart_subtotal_excl_tax":0,"template":"Magento_Tax\/checkout\/minicart\/subtotal\/totals"},"children":{"subtotal.totals.msrp":{"component":"Magento_Msrp\/js\/view\/checkout\/minicart\/subtotal\/totals","config":{"displayArea":"minicart-subtotal-hidden","template":"Magento_Msrp\/checkout\/minicart\/subtotal\/totals"}}},"component":"Magento_Tax\/js\/view\/checkout\/minicart\/subtotal\/totals"}},"component":"uiComponent","config":{"template":"Magento_Checkout\/minicart\/subtotal"}}},"component":"uiComponent","config":{"displayArea":"subtotalContainer"}},"item.renderer":{"component":"uiComponent","config":{"displayArea":"defaultRenderer","template":"Magento_Checkout\/minicart\/item\/default"},"children":{"item.image":{"component":"Magento_Catalog\/js\/view\/image","config":{"template":"Magento_Catalog\/product\/image","displayArea":"itemImage"}},"checkout.cart.item.price.sidebar":{"component":"uiComponent","config":{"template":"Magento_Checkout\/minicart\/item\/price","displayArea":"priceSidebar"}}}},"extra_info":{"component":"uiComponent","config":{"displayArea":"extraInfo"}},"promotion":{"component":"uiComponent","config":{"displayArea":"promotion"}}},"config":{"itemRenderer":{"default":"defaultRenderer","simple":"defaultRenderer","virtual":"defaultRenderer"},"template":"Magento_Checkout\/minicart\/content"},"component":"Magento_Checkout\/js\/view\/minicart"}},"types":[]}        },
        "*": {
            "Magento_Ui/js/block-loader": "https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/images/loader-1.gif"
        }
    }
    </script>
<script type="text/javascript">
        //<![CDATA[
        requirejs(['jquery'], function(jQuery) {
            jQuery(function($) {
                // If header is in mobile mode, remove "display: block". See #10.
                $(document).on('dropdowndialogcreate', function(event) {
                    if ($('#header-container').hasClass('header-mobile'))
                    {
                        $('#header-cart').css('display', '');
                    }
                });
                $(document).on('dropdowndialogbeforeclose', function(event) {
                    $('#header-cart').data('mage-dropdownDialog').opener = $(document.body); // See #5
                });
            });
        }); //end: requirejs
        //]]>
    </script>
</div>
<div class="skip-content skip-content--style" id="header-nav">
<div id="nav-marker-mobile"></div>
</div>
<div id="search-marker-mobile"></div>
<div class="skip-content skip-content--style" id="header-search">
<div class="block block-search search-wrapper" id="block-search"> <div class="block block-title"><strong>Search</strong></div>
<div class="block block-content">
<form action="https://www.simpsonspremium.com/catalogsearch/result/" class="form minisearch" id="search_mini_form" method="get">
<div class="field search">
<div class="control">
<input aria-autocomplete="both" aria-haspopup="false" autocomplete="off" class="input-text" data-mage-init='{"quickSearch":{
                                "formSelector":"#search_mini_form",
                                "url":"https://www.simpsonspremium.com/search/ajax/suggest/",
                                "destinationSelector":"#search_autocomplete"}
                           }' id="search" maxlength="128" name="q" placeholder="Search For Products" role="combobox" type="text" value=""/>
<div class="search-autocomplete" id="search_autocomplete"></div>
</div>
</div>
<div class="actions">
<button class="action search" id="action-search" title="Search" type="submit">
<span class="icon ic ic-search ib ib-square ib-hover"></span>
</button>
</div>
</form>
</div>
</div> <!-- end: block-search -->
</div>
<div id="account-links-marker-mobile"></div>
<div class="account-links top-links skip-content skip-content--style grid12-9 no-gutter" id="header-account">
<ul class="header links"><li class="nav item"><a href="https://www.simpsonspremium.com/stockists/">Stockists</a></li><li class="nav item"><a href="https://www.simpsonspremium.com/delivery/">Delivery</a></li><li><a href="https://www.simpsonspremium.com/customer/account/">Account</a></li><li class="authorization-link" data-label="or">
<a href="https://www.simpsonspremium.com/customer/account/login/">
        Log In    </a>
</li>
<li><a href="https://www.simpsonspremium.com/customer/account/create/">Register</a></li></ul> </div>
<div class="skip-links-clearer clearer"></div>
</div> <!-- end: skip-links-wrapper -->
</div> <!-- end: inner-container -->
</div> <!-- end: header-m-primary -->
</div> <!-- end: header-m-primary-container -->
</div> <!-- end: header-m-container -->
<div class="header-primary-container">
<div class="header-primary header container">
<div class="inner-container">
<div class="hp-blocks-holder">
<!-- Left column -->
<div class="hp-block left-column grid12-4">
<div class="item"><div class="logo-wrapper logo-wrapper--regular">
<div class="logo">
<a href="https://www.simpsonspremium.com/" title="Simpsons Premium">
<strong>Simpsons Premium</strong>
<img alt="Simpsons Premium" src="https://www.simpsonspremium.com/pub/media/logo/default/logo.png"/>
</a>
</div>
</div></div>
</div> <!-- end: left column -->
<div class="hp-block central-column grid12-3">
<!-- empty dummy block !-->
</div>
<!-- Right column -->
<div class="hp-block right-column grid12-5">
<div id="account-links-marker-regular"></div> <div class="grid12-3" id="lang-switcher-wrapper-regular">
<div class="dropdown-block filled-heading dropdown-block-trigger lang-switcher" id="switcher-language">
<div class="dropdown-heading">
<span class="inner-heading view-uk" title="Language">
<span class="dropdown-icon"> </span>
<span class="label">English</span>
<span class="caret"></span>
</span>
</div>
<ul class="dropdown-content">
<li class="view-it switcher-option">
<a href="/it"><span class="label dropdown-icon"> </span>Italian</a>
</li>
<li class="view-fr switcher-option">
<a href="/fr"><span class="label dropdown-icon"> </span>French</a>
</li>
<li class="view-nl switcher-option">
<a href="/nl"><span class="label dropdown-icon"> </span>Dutch</a>
</li>
<li class="view-be switcher-option">
<a href="/be"><span class="label dropdown-icon"> </span>Belgian</a>
</li>
</ul>
</div>
</div> <div class="grid12-12 no-gutter" id="user-menu-wrapper-regular">
<div class="user-menu" id="user-menu">
<div id="mini-cart-marker-regular"></div>
<a href="/checkout" id="user-menu-checkout">
<button class="action">Checkout <i class="caret-right"> </i></button>
</a>
</div> <!-- end: user-menu -->
</div>
</div> <!-- end: right column -->
</div> <!-- end: hp-blocks-holder -->
</div> <!-- end: inner-container -->
</div> <!-- end: header-primary -->
</div> <!-- end: header-primary-container -->
<div class="sticky-container">
<div class="nav-container skip-content">
<div class="nav clearer">
<div class="inner-container container"><div class="navi-wrapper">
<ul class="nav-holders-wrapper right-column">
<li class="nav-item level0 level-top nav-holder" id="nav-holder1"><div id="search-marker-regular"></div></li>
<li class="nav-item level0 level-top nav-holder" id="nav-holder2"></li>
<li class="nav-item level0 level-top nav-holder" id="nav-holder3"></li>
</ul>
<div class="mobnav-trigger menu-trigger" id="mobnav-trigger">
<div class="menu-trigger-inner">
<span class="trigger-icon"><span class="line"></span><span class="line"></span><span class="line"></span></span>
<span class="label">Menu</span>
</div>
</div>
<div id="nav-marker-regular"></div>
<nav class="navi nav-regular opt-fx-fade-inout opt-sb0 opt-sob opt-hide480 with-bullets" id="mainmenu">
<ul>
<li class="nav-item nav-item--home level0 level-top">
<a class="level-top" href="https://www.simpsonspremium.com/" title="Go To Homepage"><span>Home</span></a>
</li>
<li class="nav-item level0 nav-1 level-top first nav-item--parent mega nav-item--only-blocks parent"><a class="level-top" href="https://www.simpsonspremium.com/dogs"><span>Dogs</span><span class="caret"></span></a><span class="opener"></span><div class="nav-panel--dropdown nav-panel full-width"><div class="nav-panel-inner"><div class="nav-block nav-block--left std grid12-6"><ul class="level0 nav-submenu nav-submenu--mega dd-itemgrid dd-itemgrid-2col">
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs">View all Products</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/adult">Adult dog food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/8020">80/20 dog food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/puppy">Puppy dog food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/rice">Rice Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/senior">Senior Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/sensitive">Sensitive Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/6040">60/40 Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/wet">Wet Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/grain-free-dog-food">Grain Free Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/high-meat-dog-food">High Meat Dog Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/dogs/dry-dog-food">Dry Dog Food</a></li>
</ul></div><div class="nav-block nav-block--right std grid12-6"><div class="grid12-6 menu-block menu-block-l">
<div class="menu-block-img"><a href="https://www.simpsonspremium.com/dogs/puppy"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_dogs_puppy.png"/></a></div>
<div class="menu-block-link"><a href="https://www.simpsonspremium.com/dogs/puppy">Our Puppy Food Range<span>&gt;</span></a></div>
</div>
<div class="grid12-6 menu-block menu-block-r">
<div class="menu-block-img"><a href="https://www.simpsonspremium.com/dogs/adult"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_dogs_adult.png"/></a></div>
<div class="menu-block-link"><a href="https://www.simpsonspremium.com/dogs/adult">Our Adult Food Range<span>&gt;</span></a></div>
</div></div></div></div></li><li class="nav-item level0 nav-2 level-top nav-item--parent mega nav-item--only-blocks parent"><a class="level-top" href="https://www.simpsonspremium.com/treats"><span>Treats</span><span class="caret"></span></a><span class="opener"></span><div class="nav-panel--dropdown nav-panel" style="width:50%;"><div class="nav-panel-inner"><div class="nav-block nav-block--left std grid12-6"><ul class="level0 nav-submenu nav-submenu--mega dd-itemgrid dd-itemgrid-1col">
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/treats">View all Products</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/treats/air-dried-treats">Air Dried Treats</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/treats/fish-skin-treats">Fish Skin Treats</a></li>
</ul></div><div class="nav-block nav-block--right std grid12-6"><div class="grid12-12 menu-block">
<div class="menu-block-img"><a href="https://www.simpsonspremium.com/treats"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_treats.jpg"/></a></div>
<div class="menu-block-link"><a href="https://www.simpsonspremium.com/treats">Treats for Dogs<span>&gt;</span></a></div>
</div></div></div></div></li><li class="nav-item level0 nav-3 level-top last nav-item--parent mega nav-item--only-blocks parent"><a class="level-top" href="https://www.simpsonspremium.com/cats"><span>Cats</span><span class="caret"></span></a><span class="opener"></span><div class="nav-panel--dropdown nav-panel full-width"><div class="nav-panel-inner"><div class="nav-block nav-block--left std grid12-6"><ul class="level0 nav-submenu nav-submenu--mega dd-itemgrid dd-itemgrid-2col">
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/cats">View all Products</a></li></ul></div></div></div></li></ul></nav></div></div></div></div></div></div></div></div></div></body></html>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/cats/adult">Adult Cat Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/cats/kitten">Kitten Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/cats/dry">Dry Cat Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="https://www.simpsonspremium.com/cats/wet">Wet Cat Food</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="/cats/cat-variety-packs" target="_self">Variety Packs</a></li>
<div class="nav-block nav-block--right std grid12-6"><div class="grid12-6 menu-block menu-block-l">
<div class="menu-block-img"><a href="https://www.simpsonspremium.com/cats/kitten"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_cats_kitten.jpg"/></a></div>
<div class="menu-block-link"><a href="https://www.simpsonspremium.com/cats/kitten">Our Kitten Food Range<span>&gt;</span></a></div>
</div>
<div class="grid12-6 menu-block menu-block-l">
<div class="menu-block-img"><a href="https://www.simpsonspremium.com/cats/adult"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_cats_adult.jpg"/></a></div>
<div class="menu-block-link"><a href="https://www.simpsonspremium.com/cats/adult">Our Adult Food Range<span>&gt;</span></a></div></div></div> <li class="nav-item nav-4 level0 level-top">
<a class="level-top" href="/loyalty-scheme">Loyalty Scheme</a>
</li>
<li class="nav-item level0 nav-5 level-top last nav-item--parent classic nav-item--only-subcategories parent">
<a class="level-top" href="/faqs"><span>Useful Info</span><span class="caret"></span></a>
<span class="opener"></span>
<ul class="level0 nav-submenu nav-panel--dropdown nav-panel">
<li class="nav-item level1 nav-1-1 first">
<a href="/ingredients"><span>Ingredients</span></a>
</li>
<li class="nav-item level1 nav-1-2">
<a href="/breeders-club"><span>BREEDERS</span></a>
</li>
<li class="nav-item level1 nav-1-3">
<a href="/about-us"><span>ABOUT</span></a>
</li>
<li class="nav-item level1 nav-1-4">
<a href="/faqs"><span>FAQ's</span></a>
</li>
</ul> </li>
<li class="nav-item level0 nav-5 level-top last nav-item--parent classic nav-item--only-subcategories parent nav-item--account">
<a class="level-top" href="/customer/account"><span>Your Account</span><span class="caret"></span></a>
<span class="opener"></span>
<ul class="level0 nav-submenu nav-panel--dropdown nav-panel">
<li class="nav-item level1 nav-1-0 view-all"><a href="/customer/account/login">Login</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="/customer/account">My Account</a></li>
<li class="nav-item level1 nav-1-0 view-all"><a href="/customer/account/create">Register</a></li>
</ul>
</li>
<li class="nav-item level0 nav-6 level-top last nav-item--parent classic nav-item--only-subcategories parent">
<a class="level-top" href="/advice-centre"><span>Advice Centre</span><span class="caret"></span></a>
<span class="opener"></span>
<div class="nav-panel--dropdown nav-panel full-width">
<div class="nav-panel-inner advice-centre-menu">
<div class="nav-block--left grid12-6">
<div class="ac-inner clearfix">
<div class="ac-heading"><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/">dogs</a></div>
<div class="grid12-7 ac-info">
<a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_advice_dogs.jpg"/></a>
<div class="ac-dtl">
<div class="ac-name"><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/">Advice For Dogs</a></div>
<p>Find out everything you need to know about your new dog or puppy in our advice centre.</p>
</div>
</div>
<div class="grid12-5 ac-links">
<ul>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=dog-breeds/">Dog Breeds</a></li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=dog-health-nutrition/">Health &amp; Nutrition</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=puppy-guide/">Puppy Guide</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=dog-training-behaviour/">Training &amp; Behaviour</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=infographics/">Infographics</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/?id=other/">Other</a></li>
<li><button><a href="https://www.simpsonspremium.com/advice-centre/advice-for-dogs/">view all </a></button></li>
</ul>
</div>
</div> </div>
<div class="nav-block--right grid12-6">
<div class="ac-inner clearfix">
<div class="ac-heading"><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/">cats</a></div>
<div class="grid12-7 ac-info">
<a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/"><img alt="" src="https://www.simpsonspremium.com/pub/media/wysiwyg/menu_advice_cats.jpg"/></a>
<div class="ac-dtl">
<div class="ac-name"><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/">Advice For Cats</a></div>
<p>Browse through our categories and learn how to look after a cat or kitten in no time.</p>
</div>
</div>
<div class="grid12-5 ac-links">
<ul>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=cat-breeds/">Cat Breeds</a></li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=cat-health-nutrition/">Health &amp; Nutrition</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=kitten-guide/">Kitten Guide</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=cat-training-behaviour/">Training &amp; Behaviour</a></li>
<li> <a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=infographics/">Infographics</a> </li>
<li><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/?id=other/">Other</a></li>
<li><button><a href="https://www.simpsonspremium.com/advice-centre/advice-for-cats/">view all</a></button> </li></ul></div></div></div></div></div></li>


 



<li class="nav-item nav-6 level0 level-top">
<a class="level-top" href="/contact-us">Contact</a>
</li>


 <!-- end: navi-wrapper -->
<script type="text/javascript">
//<![CDATA[

requirejs(['jquery', 'ultramegamenu'], function(jQuery, ultramegamenu) {

    var topMenuContainer = jQuery('#mainmenu');
    var topMenuSettings = {
        mobileMenuThreshold: 992        , isVerticalLayout: false        , mobnavTriggerSelector: '#mobnav-trigger'
        , vertnavTriggerSelector: '#vertnav-trigger'
        , mode: 0                , initVerticalMenuCollapsed: true        , outermostContainer: jQuery('.hp-blocks-holder')        , fullWidthDdContainer: jQuery('.hp-blocks-holder')    };
    var theTopMenu = topMenuContainer.ultramegamenu(topMenuSettings).data("infortis-ultramegamenu");
    theTopMenu.enableDropdowns();

        
        jQuery(function($) {

            var itemsList = topMenuContainer.children('ul');

            // Non-clickable links
            itemsList.on('click', '.no-click', function(e) {
                e.preventDefault();
            });

        }); //end: on document ready

        jQuery(window).on("load", function() {

            var menubar = topMenuContainer;
            var isTouchDevice = ('ontouchstart' in window) || (navigator.msMaxTouchPoints > 0);
            if (isTouchDevice)
            {
                menubar.on('click', 'a', function(e) {

                    var link = jQuery(this);
                    if (!menubar.hasClass('nav-mobile') && link.parent().hasClass('nav-item--parent'))
                    {
                        if (!link.hasClass('ready'))
                        {
                            e.preventDefault();
                            menubar.find('.ready').removeClass('ready');
                            link.parents('li').children('a').addClass('ready');
                        }
                    }

                }); //end: on click
            } //end: if isTouchDevice

        }); //end: on load

    
}); //end: requirejs

//]]>
require([
        'jquery',
        'js/jquery-ui'
      ], function ($, script) {

        $("li.nav-item.mega.parent > a").each(function(){
            $(this).addClass("top-level-link");
        });
        $(".nav-item.level1.first").each(function(){
            var parent = $(this).parent('ul').closest('li.level0').children('a.top-level-link');   
            var copy = parent.clone().wrap('<li></li>');
            $(this).before(copy[0]);
            var parent = null;
            var copy = null;
        });
        $(".nav-submenu a.top-level-link").each(function(){
            $(this).wrap('<li class="nav-item level1 nav-1-0 view-all"></li>');
            $(this).text("View All Products");
        });          

    });



</script>



<div class="ups-container">
<div class="container clearfix" id="ups">
<div class="ups-column ups-1">
<div class="ups-inner">
<div class="image-box"><a href="/loyalty-scheme"><img alt="Loyaty Scheme" id="ups-img-1" src="https://www.simpsonspremium.com/pub/media/wysiwyg/SP_icon_4_1.png"/></a></div>
<div class="text-box"><a href="/loyalty-scheme">5% Loyalty Scheme</a></div>
</div>
</div>
<div class="ups-column ups-2">
<div class="ups-inner">
<div class="image-box"><a href="/delivery"><img alt="Free delivery on Order over £25" id="ups-img-2" src="https://www.simpsonspremium.com/pub/media/wysiwyg/SP_icon_3_1.png"/></a></div>
<div class="text-box"><a href="/delivery">Free delivery on Orders over £30*</a><span>*Terms and Conditions Apply</span></div>
</div>
</div>
<div class="ups-column ups-3">
<div class="ups-inner">
<div class="image-box"><a href="/treats"><img alt="New treats Range" id="ups-img-3" src="https://www.simpsonspremium.com/pub/media/wysiwyg/Dog_bone_1.png"/></a></div>
<div class="text-box"><a href="/treats">New treats range</a></div>
</div>
</div>
</div>
</div>

 <!-- end: header-container3 -->
 <!-- end: header-container2 -->
 <!-- end: header-container -->
<script type="text/javascript">
//<![CDATA[

requirejs(['jquery', 'smartheader', 'stickyheader', 'dropdownDialog'], function(jQuery, smartheader, stickyheader, dropdownDialog) {

    var theHeaderContainer = jQuery('#header-container');

        
        //alert('header tpl, before smartheader');

        theHeaderContainer.smartheader();

    
    jQuery(function($) {

        //console.log('header tpl, on(ready), ater smartheader');
        //alert('header tpl, on(ready), ater smartheader');

        
            // Skip Links
            var skipContents = $('.skip-content');
            var skipLinks = $('.skip-link');
            var mobileHeader = $('.header-m-container .inner-container');

            skipLinks.on('click', function (event) {

                // In case skip link is "a" tag
                event.preventDefault();

                var self = $(this);
                var target = self.data('skiptarget');

                // Get target element
                var elem = $(target);

                // Check if stub is open
                var isSkipContentOpen = elem.hasClass('skip-active') ? 1 : 0;

                // Hide all stubs
                skipLinks.removeClass('skip-active');
                skipContents.removeClass('skip-active');

                // Toggle stubs
                if (isSkipContentOpen) {
                    self.removeClass('skip-active');
                } else {
                    self.addClass('skip-active');
                    elem.addClass('skip-active');
                }
            });

            // Mobile sticky header
            $(window).scroll(function(){
                if($(this).scrollTop() >= 168){
                    if (!mobileHeader.hasClass('mobile-header-sticky')) {
                        mobileHeader.addClass('mobile-header-sticky');
                        mobileHeader.css({display: 'none'});
                        mobileHeader.slideDown(250);
                    }
                }
                if($(this).scrollTop() === 0){
                    if (mobileHeader.hasClass('mobile-header-sticky')) {
                        mobileHeader.removeClass('mobile-header-sticky');

                    }
                }
            });

        
        
            var stickyHeaderSettings = {
                stickyThreshold: 992            };
            theHeaderContainer.stickyheader(stickyHeaderSettings);

        
    }); //end: on document ready

}); //end: requirejs

//]]>
</script>
<script type="text/javascript">
(function (d, t) {
  var bh = d.createElement(t), s = d.getElementsByTagName(t)[0];
  bh.type = 'text/javascript';
  bh.src = 'https://www.bugherd.com/sidebarv2.js?apikey=x2ngakqwrb3w9djhshc3fw';
  s.parentNode.insertBefore(bh, s);
  })(document, 'script');
</script><div class="main-container"><div class="main container"><div class="inner-container"><main class="page-main" id="maincontent"><a id="contentarea" tabindex="-1"></a>
<div class="page messages"><div data-placeholder="messages"></div>
<div data-bind="scope: 'messages'">
<!-- ko if: cookieMessages && cookieMessages.length > 0 -->
<div class="messages" data-bind="foreach: { data: cookieMessages, as: 'message' }" role="alert">
<div data-bind="attr: {
            class: 'message-' + message.type + ' ' + message.type + ' message',
            'data-ui-id': 'message-' + message.type
        }">
<div data-bind="html: message.text"></div>
</div>
</div>
<!-- /ko -->
<!-- ko if: messages().messages && messages().messages.length > 0 -->
<div class="messages" data-bind="foreach: { data: messages().messages, as: 'message' }" role="alert">
<div data-bind="attr: {
            class: 'message-' + message.type + ' ' + message.type + ' message',
            'data-ui-id': 'message-' + message.type
        }">
<div data-bind="html: message.text"></div>
</div>
</div>
<!-- /ko -->
</div>
<script type="text/x-magento-init">
    {
        "*": {
            "Magento_Ui/js/core/app": {
                "components": {
                        "messages": {
                            "component": "Magento_Theme/js/view/messages"
                        }
                    }
                }
            }
    }
</script>
</div><div class="columns"><div class="column main"><input name="form_key" type="hidden" value="3J3FwYgCe7xRXlZB"/><div data-bind="scope:'authenticationPopup'" id="authenticationPopup" style="display: none;">
<script>
        window.authenticationPopup = {"autocomplete":"off","customerRegisterUrl":"https:\/\/www.simpsonspremium.com\/customer\/account\/create\/","customerForgotPasswordUrl":"https:\/\/www.simpsonspremium.com\/customer\/account\/forgotpassword\/","baseUrl":"https:\/\/www.simpsonspremium.com\/"};
    </script>
<!-- ko template: getTemplate() --><!-- /ko -->
<script type="text/x-magento-init">
        {
            "#authenticationPopup": {
                "Magento_Ui/js/core/app": {"components":{"authenticationPopup":{"component":"Magento_Customer\/js\/view\/authentication-popup","children":{"messages":{"component":"Magento_Ui\/js\/view\/messages","displayArea":"messages"},"captcha":{"component":"Magento_Captcha\/js\/view\/checkout\/loginCaptcha","displayArea":"additional-login-form-fields","formId":"user_login","configSource":"checkout"},"amazon-button":{"component":"Amazon_Login\/js\/view\/login-button-wrapper","sortOrder":"0","displayArea":"additional-login-form-fields","config":{"tooltip":"Securely login to our website using your existing Amazon details."}}}}}}            },
            "*": {
                "Magento_Ui/js/block-loader": "https\u003A\u002F\u002Fwww.simpsonspremium.com\u002Fpub\u002Fstatic\u002Fversion1574949831\u002Ffrontend\u002FFat\u002Fsimpsons\u002Fen_GB\u002Fimages\u002Floader\u002D1.gif"
            }
        }
    </script>
</div>
<script type="text/x-magento-init">
{"*":{"Magento_Customer\/js\/section-config":{"sections":{"stores\/store\/switch":"*","directory\/currency\/switch":"*","*":["messages","location"],"customer\/account\/logout":["recently_viewed_product","recently_compared_product"],"customer\/account\/loginpost":"*","customer\/account\/createpost":"*","customer\/account\/editpost":"*","customer\/ajax\/login":["checkout-data","cart","captcha"],"catalog\/product_compare\/add":["compare-products"],"catalog\/product_compare\/remove":["compare-products"],"catalog\/product_compare\/clear":["compare-products"],"sales\/guest\/reorder":["cart"],"sales\/order\/reorder":["cart"],"checkout\/cart\/add":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/delete":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/updatepost":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/updateitemoptions":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/couponpost":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/estimatepost":["cart","magepal-gtm-jsdatalayer"],"checkout\/cart\/estimateupdatepost":["cart","magepal-gtm-jsdatalayer"],"checkout\/onepage\/saveorder":["cart","checkout-data","last-ordered-items","rewards","magepal-gtm-jsdatalayer","checkout-fields"],"checkout\/sidebar\/removeitem":["cart","magepal-gtm-jsdatalayer"],"checkout\/sidebar\/updateitemqty":["cart","magepal-gtm-jsdatalayer"],"rest\/*\/v1\/carts\/*\/payment-information":["cart","checkout-data","last-ordered-items","instant-purchase","rewards","magepal-gtm-jsdatalayer"],"rest\/*\/v1\/guest-carts\/*\/payment-information":["cart","magepal-gtm-jsdatalayer"],"rest\/*\/v1\/guest-carts\/*\/selected-payment-method":["cart","checkout-data","magepal-gtm-jsdatalayer"],"rest\/*\/v1\/carts\/*\/selected-payment-method":["cart","checkout-data","instant-purchase","magepal-gtm-jsdatalayer"],"authorizenet\/directpost_payment\/place":["cart","checkout-data"],"customer\/address\/*":["instant-purchase"],"customer\/account\/*":["instant-purchase"],"vault\/cards\/deleteaction":["instant-purchase"],"multishipping\/checkout\/overviewpost":["cart"],"paypal\/express\/placeorder":["cart","checkout-data"],"paypal\/payflowexpress\/placeorder":["cart","checkout-data"],"paypal\/express\/onauthorization":["cart","checkout-data"],"persistent\/index\/unsetcookie":["persistent"],"review\/product\/post":["review"],"braintree\/paypal\/placeorder":["cart","checkout-data"],"wishlist\/index\/add":["wishlist"],"wishlist\/index\/remove":["wishlist"],"wishlist\/index\/updateitemoptions":["wishlist"],"wishlist\/index\/update":["wishlist"],"wishlist\/index\/cart":["wishlist","cart"],"wishlist\/index\/fromcart":["wishlist","cart"],"wishlist\/index\/allcart":["wishlist","cart"],"wishlist\/shared\/allcart":["wishlist","cart"],"wishlist\/shared\/cart":["cart"]},"clientSideSections":["checkout-data","cart-data"],"baseUrls":["https:\/\/www.simpsonspremium.com\/"]}}}</script>
<script type="text/x-magento-init">
{"*":{"Magento_Customer\/js\/customer-data":{"sectionLoadUrl":"https:\/\/www.simpsonspremium.com\/customer\/section\/load\/","expirableSectionLifetime":60,"expirableSectionNames":["cart","persistent"],"cookieLifeTime":"3600","updateSessionUrl":"https:\/\/www.simpsonspremium.com\/customer\/account\/updateSession\/"}}}</script>
<script type="text/x-magento-init">
{"*":{"Magento_Customer\/js\/invalidation-processor":{"invalidationRules":{"website-rule":{"Magento_Customer\/js\/invalidation-rules\/website-rule":{"scopeConfig":{"websiteId":1}}}}}}}</script>
<script type="text/x-magento-init">
    {
        "body": {
            "pageCache": {"url":"https:\/\/www.simpsonspremium.com\/page_cache\/block\/render\/","handles":["default","customer_group_general","cms_noroute_index","cms_page_view","cms_noroute_index_id_no-route"],"originalRequest":{"route":"cms","controller":"noroute","action":"index","uri":"\/index.py"},"versionCookieName":"private_content_version"}        }
    }
</script>
<style type="text/css">
                            .form-customer-login .g-recaptcha {
                padding-top: 15%;
            }
        
        .form-customer-login .g-recaptcha {
            margin: 0px;
        }

        #social-form-create .g-recaptcha{
            padding-top: 5%;
        }

        #social-form-password-forget .g-recaptcha{
            padding-top: 5%;
        }

        .onestepcheckout-index-index .form-login .g-recaptcha .grecaptcha-badge {
            position: unset !important;
        }

        .checkout-index-index form[data-role=login] .g-recaptcha .grecaptcha-badge {
            position: unset !important;
        }

        #mpageverify-form .g-recaptcha {
            margin-left: 30%;
        }

        .g-recaptcha {
            margin-top: 15px;
            margin-bottom: 15px;
        }
    </style>
<script type="text/x-magento-init">
        {
            "*": {
                "Mageplaza_GoogleRecaptcha/js/captcha": {
                    "key": "6LctsIkUAAAAAFld1ZFxEULF2k8LO7Kl5opagr4S",
                    "language": "en-GB",
                    "position": "bottomright",
                    "theme": "light",
                    "forms": ["#social-login-form","#social-form-create","#social-form-password-forget","#contact-form"],
                    "type": "invisible",
                    "size": ""
                }
            }
        }
    </script>
<div class="page-title pnf">
<h1>LOST PAGE!</h1>
</div>
<div class="no-route grid12-6 no-left-gutter">
<div class="no-route sec-title">Our beloved page has gone missing.</div>
<p>Very friendly, loves other animals and people. Answers to the name 404. If found, please return home.</p>
<a class="btn btn--secondary" href="https://www.simpsonspremium.com/"><button class="action">RETURN TO HOME</button></a></div>
<div class="no-route grid12-6 no-right-gutter"><img alt="dog" height="400px" src="https://www.simpsonspremium.com/pub/media/wysiwyg/dog1_1.jpg" width="400px"/></div>
<div class="no-route h2">Get back to the right page below</div>
<div class="post-list">
<ul class="grid12-12 no-gutter" id="post-list">
<li class="pageval item grid12-4"><img alt="dogeating" height="400px" src="https://www.simpsonspremium.com/pub/media/wysiwyg/dog.jpg" width="400px"/>
<div class="no-route h3">PREMIUM DOG FOOD</div>
<p>It doesn't matter if your dogs are show dogs, working dogs or pets first and foremost , they deserve the best at mealtimes. Browse our premium dog food selection today.</p>
<button class="action"><a class="btn" href="/dogs">SHOP NOW</a></button></li>
<li class="pageval item grid12-4"><img alt="dogeating" class="whiteborder" height="400px" src="https://www.simpsonspremium.com/pub/media/wysiwyg/cat.jpg" width="400px"/>
<div class="no-route h3">PREMIUM CAT FOOD</div>
<p>Your cats - whether they're moggies or pedigrees - need food that is rich in meal and low in carbohydrates. Browse our premium cat food selection today.</p>
<button class="action"><a class="btn" href="/cats">SHOP NOW</a></button></li>
<li class="pageval item grid12-4"><img alt="dogeating" class="whiteborder" height="400px" src="https://www.simpsonspremium.com/pub/media/wysiwyg/advice.jpg" width="400px"/>
<div class="no-route h3">SIMPSONS ADVICE CENTRE</div>
<p>The Simpsons Advice Center is the best place to get advice about your cat or dog, and includes tips on nutrition, health, training and information about specific breeds.</p>
<button class="action"><a class="btn" href="/advice-centre">READ MORE</a></button></li>
</ul>
</div></div></div></main></div></div></div><div class="footer-container page-footer">
<div class="footer-container2">
<div class="footer-container3">
<div class="footer-top-container section-container">
<div class="footer-top footer container">
<div class="inner-container links-wrapper-separators grid12-6">
<h4>Product Ranges</h4>
<div class="grid12-4 footer-links-col"><ul class="footer-ranges-1stColumn">
<li>Dogs</li>
<li><a href="/dogs/adult">Adult Dog Food</a></li>
<li><a href="/dogs/sensitive">Sensitive Dog Food</a></li>
<li><a href="/dogs/8020">80/20 Dog Food</a></li>
<li><a href="/dogs/6040">60/40 Dog Food</a></li>
<li><a href="/dogs/wet">Wet Dog Food</a></li>
<li><a href="/dogs/puppy">Puppy Dog Food</a></li>
</ul></div>
<div class="grid12-4 footer-links-col"><ul class="footer-ranges-1stColumn">
<li>Dogs</li>
<li><a href="/dogs/grain-free-dog-food">Grain Free Dog Food</a></li>
<li><a href="/dogs/rice">Rice Dog Food</a></li>
<li><a href="/dogs/high-meat-dog-food">High Meat Dog Food</a></li>
<li><a href="/dogs/senior">Senior Dog Food</a></li>
<li><a href="/dogs/dry-dog-food">Dry Dog Food</a></li>
</ul></div>
<div class="grid12-4 footer-links-col"><ul>
<li>Cats</li>
<li><a href="/cats/adult">Adult Cat Food</a></li>
<li><a href="/cats/kitten">Kitten Food</a></li>
<li><a href="/cats/dry">Dry Cat Food</a></li>
<li><a href="/cats/wet">Wet Cat Food</a></li>
</ul></div>
</div> <!-- end: inner-container -->
<div class="inner-container links-wrapper-separators footer-social grid12-3">
<h4>Connect</h4>
<a href="https://www.facebook.com/simpsonspremium/" target="_blank"><i class="fa fa-facebook"></i><span><span class="res-label">Connect with us on </span>Facebook</span>
</a>
<a href="https://twitter.com/simpsonspetfood" target="_blank"><i class="fa fa-twitter"></i><span class="res-label">Follow us on </span>Twitter</a></div></div></div></div></div></div>

<div class="qa_wrapper">
<img alt="The Queen's Award For Enterprise: International Trade 2018" src="https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/images/qa_emblem.png" title="The Queen\'s Award For Enterprise: International Trade 2018"/>
</div>

<div class="inner-container links-wrapper-separators grid12-3 contact-footer-info">
<div class="contact-footer-info-inner">
<h4>Contact</h4>
<ul class="contact-details-list">
<li>Simpsons Premium Ltd</li>
<li>Unit 17b Cinnamon Brow, Makerfield Way</li>
<li>Ince, Wigan</li>
<li>WN2 2PR</li>
</ul>
<p><i class="fa fa-phone"></i>
                01942 246436</p>
<p> Mon-Fri 9.30-5.00</p>
</div>
<img alt="Secured by SagePay" src="https://www.simpsonspremium.com/pub/static/version1574949831/frontend/Fat/simpsons/en_GB/images/secured_by.jpg" title="Secured by SagePay"/>
</div>
 <!-- end: footer-top -->

<div class="footer-bottom-container section-container">
<div class="footer-bottom footer container">
<div class="inner-container">
<div class="item item-left">
<div class="footer-bottom-links"><ul>
<li><a href="/terms-of-use">Terms of Use</a></li>
<li><a href="/privacy-policy">Privacy Policy</a></li>
<li><a href="/cookies-policy">Cookies Policy</a></li>
<li><a href="/sitemap/">Site Map</a></li>
</ul> </div>
</div>
<div class="item item-right">
<div class="footer-copyright"><small class="copyright">
<span>© Simpsons Premium 2018. All rights reserved.</span>
</small>
</div>
</div>
</div> <!-- end: inner-container -->
</div> <!-- end: footer-bottom -->
</div>
<a class="ic ic-up" href="#top" id="scroll-to-top"></a>
 <!-- end: footer-container3 -->
 <!-- end: footer-container2 -->
 <!-- end: footer-container -->
<script type="text/x-magento-init">
        {
            "*": {
                "Magento_Ui/js/core/app": {
                    "components": {
                        "storage-manager": {
                            "component": "Magento_Catalog/js/storage-manager",
                            "appendTo": "",
                            "storagesConfiguration" :
                                         {"recently_viewed_product":{"requestConfig":{"syncUrl":"https:\/\/www.simpsonspremium.com\/catalog\/product\/frontend_action_synchronize\/"},"lifetime":"1000","allowToSendRequest":null},"recently_compared_product":{"requestConfig":{"syncUrl":"https:\/\/www.simpsonspremium.com\/catalog\/product\/frontend_action_synchronize\/"},"lifetime":"1000","allowToSendRequest":null},"product_data_storage":{"updateRequestConfig":{"url":"https:\/\/www.simpsonspremium.com\/rest\/uk\/V1\/products-render-info"},"allowToSendRequest":null}}                        }
                    }
                }
            }
        }
</script>
<script type="text/javascript">
//<![CDATA[

requirejs(['jquery'], function(jQuery) {

        
        var gridItemsEqualHeightApplied = false;
        function setGridItemsEqualHeight()
        {
            var bottomMinSpace = 20; // Minimum space below the button at the bottom of the item
            var gridItemMaxHeight = 0;
            var gridItemMaxPaddingBottom = 0;
            var $listContainer = jQuery('.category-products-grid');
            var $listItems = $listContainer.children('ol').children();
            var centered = $listContainer.hasClass('centered');
            var $row = jQuery();

            $listItems.each(function() {

                var $item = jQuery(this);
                var $actionsBlock = $item.find('.actions');

                // Check if first item in a row
                if ($item.css("clear") == "left")
                {
                    // If row not empty, apply the max values to all items in a row
                    if ($row.length)
                    {
                        $row.css({
                            "height": gridItemMaxHeight + "px", 
                            "padding-bottom": gridItemMaxPaddingBottom + "px"
                        });

                        // Reset the row collection and reset the max values
                        $row = jQuery();
                        gridItemMaxHeight = 0;
                        gridItemMaxPaddingBottom = 0;
                    }
                }
                $row = $row.add($item);

                $item.css("height", "auto");                 gridItemMaxHeight = Math.max(gridItemMaxHeight, $item.height());

                // Use constant bottomMinSpace as bottom offset for the actions container
                $actionsBlock.css("bottom", bottomMinSpace + "px");

                // Align button to the center
                if (centered)
                {
                    var objectWidth = $actionsBlock.width();
                    var availableWidth = $item.width();
                    var space = availableWidth - objectWidth;
                    var leftOffset = ~~(space / 2);
                    $actionsBlock.css("padding-left", leftOffset + "px");                 }

                // Set bottom padding wich equals to: actions container height + bottomMinSpace
                var currentPaddingBottom = bottomMinSpace + $actionsBlock.innerHeight();
                gridItemMaxPaddingBottom = Math.max(gridItemMaxPaddingBottom, currentPaddingBottom);

                $item.css("padding-bottom", currentPaddingBottom + "px"); // TODO: possibly redundant. Padding will be applied for the entire row.

            });

            // Apply the max values to all items in the last row
            if ($row.length)
            {
                $row.css({
                    "height": gridItemMaxHeight + "px", 
                    "padding-bottom": gridItemMaxPaddingBottom + "px"
                });
            }
            
            gridItemsEqualHeightApplied = true;

        }

    


    jQuery(function($) {



        // Products grid: equal height of items
        
            $(document).on('last-swatch-found', function(e) {
                setGridItemsEqualHeight();
                $(document).off('last-swatch-found');
                e.stopPropagation();
            });

            setGridItemsEqualHeight();
            
        


                
            var startHeight;
            var startPaddingBottom;
            $('.category-products-grid').on('mouseenter', '.item', function() {

                    var $item = $(this);

                                                        if ($(window).width() >= 320)
                    {
                
                                            if (gridItemsEqualHeightApplied === false)
                        {
                            return false;
                        }
                    
                    var bottomMinSpace = 20;
                    var paddingBottom2 = 0;
                    var $actionsBlock = $item.find('.actions');

                    startHeight = $item.height();
                    startPaddingBottom = parseInt($item.css("padding-bottom"));

                    $item.css("height", "auto"); // Reset height
                    $item.find(".display-onhover").fadeIn(400, "easeOutCubic"); // Show elements visible on hover
                    var h2 = $item.height();
                    
                    // -------------------------------------------------------------------------
                    // Compare start padding with new on-hover padding, calculate the difference

                    // Get actions height and calculate new padding
                    // Calculate new bottom padding wich equals to: actions container height + bottomMinSpace
                    paddingBottom2 = bottomMinSpace + $actionsBlock.innerHeight();

                    // Calculate difference between start padding and new padding
                    var paddingBottomDiff = paddingBottom2 - startPaddingBottom;

                    // Apply only if new padding is larger than start padding
                    if (paddingBottomDiff > 0)
                    {
                        $item.css("padding-bottom", paddingBottom2 + "px");
                    }

                    // -------------------------------------------------------------------------
                    // Compare start height with new (on-hover) height, calculate the difference.
                    // Important: new height includes difference between start padding and new padding
                    var diff = 0;
                    if (h2 < startHeight)
                    {
                        $item.height(startHeight);
                    }
                    else
                    {
                        $item.height(h2);
                        diff = h2 - startHeight;
                        if (paddingBottomDiff > 0)
                        {
                            diff += paddingBottomDiff;
                        }
                    }
                    
                    // -------------------------------------------------------------------------
                    // Apply height difference as nagative margin, but only if new height
                    // is larger than start height.
                    if (diff > 0)
                    {
                        $item.css("margin-bottom", "-" + diff + "px");
                    }

                                    }                                 
            }).on('mouseleave', '.item', function() {

                    var $item = $(this);

                                                    if ($(window).width() >= 320)
                    {
                
                    // Clean up
                    $item.find(".display-onhover").stop(true).hide();
                    $item.css("margin-bottom", "");

                                                                $item.height(startHeight);
                        $item.css("padding-bottom", startPaddingBottom);
                    
                                    }                                 
            });
        
        


                $('.items-grid, .products-list').on('mouseenter', '.product-item-img', function() {
            $(this).find(".alt-img").fadeIn(400, "easeOutCubic");
        }).on('mouseleave', '.product-item-img', function() {
            $(this).find(".alt-img").stop(true).fadeOut(400, "easeOutCubic");
        });



                $('.fade-on-hover').on('mouseenter', function() {
            $(this).animate({opacity: 0.75}, 300, 'easeInOutCubic');
        }).on('mouseleave', function() {
            $(this).stop(true).animate({opacity: 1}, 300, 'easeInOutCubic');
        });



        // Drop-down
        var ddBlockSelector = '.dropdown-block-trigger'; // '.dropdown-block';
        var ddOpenTimeout;
        var dMenuPosTimeout;
        var DD_DELAY_IN = 200;
        var DD_DELAY_OUT = 0;
        var DD_ANIMATION_IN = 0;
        var DD_ANIMATION_OUT = 0;

        $(document).on('mouseenter touchstart', ddBlockSelector, function(e) {

            var dd = $(this);
            var ddHeading = dd.children('.dropdown-heading');
            var ddContent = dd.children('.dropdown-content');

            // If dd is not opened yet (or not initialized yet)
            var isDdOpened = dd.data('ddOpened');
            if (isDdOpened === false || isDdOpened === undefined)
            {
                // Clear old position of dd menu
                ddContent.css("left", "");
                ddContent.css("right", "");

                // Show dd menu
                clearTimeout(ddOpenTimeout);
                ddOpenTimeout = setTimeout(function() {
                    
                    dd.addClass('open');

                    // Set dd open flag
                    dd.data('ddOpened', true);
                    
                }, DD_DELAY_IN);

                ddContent.stop(true, true).delay(DD_DELAY_IN).fadeIn(DD_ANIMATION_IN, "easeOutCubic");
                
                // Set new position of dd menu.
                // This code is delayed the same amount of time as dd animation.
                clearTimeout(dMenuPosTimeout);
                dMenuPosTimeout = setTimeout(function() {

                    if (ddContent.offset().left < 0)
                    {
                        var space = dd.offset().left; // Space available on the left of dd
                        ddContent.css("left", (-1)*space);
                        ddContent.css("right", "auto");
                    }
                
                }, DD_DELAY_IN);

            } // end: dd is not opened yet

        }).on('mouseleave', ddBlockSelector, function(e) {

            var dd = $(this);
            var ddContent = dd.children('.dropdown-content');

            clearTimeout(ddOpenTimeout); // Clear, to close dd on mouseleave
            ddContent.stop(true, true).delay(DD_DELAY_OUT).fadeOut(DD_ANIMATION_OUT, "easeInCubic");
            if (ddContent.is(":hidden"))
            {
                ddContent.hide();
            }
            dd.removeClass('open');

            // Clear dd open flag
            dd.data('ddOpened', false);

            // After hiding, clear the click event flag
            dd.data('ddClickIntercepted', false);

        }).on('click', ddBlockSelector, function(e) {

            var dd = $(this);
            var ddHeading = dd.children('.dropdown-heading');
            var ddContent = dd.children('.dropdown-content');

            // Only if the heading was clicked
            if ($.contains(ddHeading[0], e.target) || ddHeading.is(e.target))
            {
                // Only after the first click already happened, the second click can close the dropdown
                if (dd.data('ddClickIntercepted'))
                {
                    if (dd.hasClass('open'))
                    {
                        clearTimeout(ddOpenTimeout); // Clear, to close dd on mouseleave
                        ddContent.stop(true, true).delay(DD_DELAY_OUT).fadeOut(DD_ANIMATION_OUT, "easeInCubic");
                        if (ddContent.is(":hidden"))
                        {
                            ddContent.hide();
                        }
                        dd.removeClass('open');

                        // Clear dd open flag
                        dd.data('ddOpened', false);

                        // After hiding, clear the click event flag
                        dd.data('ddClickIntercepted', false);
                    }
                }
                else 
                {
                    // Set the click event flag
                    dd.data('ddClickIntercepted', true);
                }
            }

        });



                var windowScroll_t;
        $(window).scroll(function(){
            
            clearTimeout(windowScroll_t);
            windowScroll_t = setTimeout(function() {
                                        
                if ($(this).scrollTop() > 100)
                {
                    $('#scroll-to-top').fadeIn();
                }
                else
                {
                    $('#scroll-to-top').fadeOut();
                }
            
            }, 500);
            
        });
        
        $('#scroll-to-top').click(function(){
            $("html, body").animate({scrollTop: 0}, 600, "easeOutCubic");
            return false;
        });



                var dResize = {

            winWidth : 0
            , winHeight : 0
            , windowResizeTimeout : null

            , init : function()
            {
                dResize.winWidth = $(window).width();
                dResize.winHeight = $(window).height();
                dResize.windowResizeTimeout;

                $(window).on('resize', function(e) {
                    clearTimeout(dResize.windowResizeTimeout);
                    dResize.windowResizeTimeout = setTimeout(function() {
                        dResize.onEventResize(e);
                    }, 50);
                });
            }

            , onEventResize : function(e)
            {
                //Prevent from executing the code in IE when the window wasn't actually resized
                var winNewWidth = $(window).width();
                var winNewHeight = $(window).height();

                //Code in this condition will be executed only if window was actually resized
                if (dResize.winWidth != winNewWidth || dResize.winHeight != winNewHeight)
                {
                    //Trigger deferred resize event
                    $(window).trigger("themeResize", e);

                    //Additional code executed on deferred resize
                    dResize.onEventDeferredResize();
                }

                //Update window size variables
                dResize.winWidth = winNewWidth;
                dResize.winHeight = winNewHeight;
            }

            , onEventDeferredResize : function() //Additional code, execute after window was actually resized
            {

                //Products grid: equal height of items
                
                    setGridItemsEqualHeight();

                
            }

        }; //end: dResize

        dResize.init();



    }); //end: on document ready



    jQuery(window).on('load',function() {

                
            setGridItemsEqualHeight();

        
    }); //end: on load



}); //end: requirejs



requirejs(['jquery', 'enquire'], function(jQuery, enquire) {

    jQuery(function($) {

        // Collapsible block
        // This plugin requires a specific markup structure. The plugin expects a set of elements that it
        // will use as the toggle link. It then hides all immediately following siblings and toggles the sibling's
        // visibility when the toggle link is clicked.
        //
        // Example markup:
        // <div class="block">
        //     <div class="block-title">Trigger</div>
        //     <div class="block-content">Content that should show when </div>
        // </div>
        //
        // JS: jQuery('.block-title').toggleSingle();
        //
        // Options:
        //     destruct: defaults to false, but if true, the plugin will remove itself, display content, and remove event handlers

        jQuery.fn.toggleSingle = function (options) {

            // passing destruct: true allows
            var settings = $.extend({
                destruct: false
            }, options);

            return this.each(function () {
                if (!settings.destruct) {
                    $(this).on('click', function () {
                        $(this)
                            .next()
                            .toggleClass('no-display')
                            .parent()
                            .toggleClass('active');
                            // .toggleClass('active')
                            // .next()
                            // .toggleClass('no-display');
                    });
                    // Hide the content
                    $this = $(this);
                    if (!$this.parent().hasClass('active'))
                    {
                        $this.next().addClass('no-display');
                    }
                }
                else
                {
                    // Remove event handler so that the toggle link can no longer be used
                    $(this).off('click');
                    // Remove all classes that were added by this plugin
                    $(this)
                        .next()
                        .removeClass('no-display')
                        .parent()
                        .removeClass('active');
                }

            });

        } // end: toggleSingle

        var breakpointScreenM = 768; // The same value as Magento's breakpoint @screen__m

        // Blocks collapsing on smaller viewports
        enquire.register('(max-width: ' + (breakpointScreenM - 1) + 'px)', {
            setup: function () {
                this.toggleElements = $(
                    '.sidebar .block:not(#layered-filter-block) .block-title, ' +
                    '.mobile-collapsible .block-title'
                );
            },
            match: function () {
                this.toggleElements.toggleSingle();
            },
            unmatch: function () {
                this.toggleElements.toggleSingle({destruct: true});
            }
        });

        // Blocks collapsing on all viewports.
        // For backward compatibility exclude blocks which have both classes: "collapsible" and "mobile-collapsible" 
        $('.collapsible:not(.mobile-collapsible) .block-title').toggleSingle();

    }); //end: on document ready

}); //end: requirejs



//]]>
</script>
 


```

**Comment by Ekultek** [comment]
basic 404
